sage-ios/SAGE/SAGE/CheckinRequestTableViewCell.swift

```diff
@@ -18,12 +18,14 @@ class CheckinRequestTableViewCell: UITableViewCell {
     var checkButton = UIButton()
     var xButton = UIButton()
     
+    var checkinID: Int?
```

Comment:
ooh actually you're right --- i'm just not using the property right now because i haven't done network requests yet, but we'll need it later when we do that
idk what i was talking about, i think that was adddirector in addschools
i get the cell by doing this which isn't great either:
let cell = sender.superview!.superview as! CheckinRequestTableViewCell